telethon_examples/README.md

```diff
@@ -130,6 +130,14 @@ assumes some [`asyncio`] knowledge, but otherwise is easy to follow.
 
 ![Screenshot of the tkinter GUI][tkinter GUI]
 
+### [`payment.py`](https://raw.githubusercontent.com/LonamiWebs/Telethon/master/telethon_examples/payment.py)
+
+* Usable as: **bot**.
+* Difficulty: **medium**.
+
+This example shows you an example of making invoices (requesting payemnts) via bot without shipping.
+Uses some raw api and some raw request, which can also help you in understanding telethon itself.
```

Comment:
```suggestion

It makes use of the ["raw API"](https://tl.telethon.dev) (that is, no friendly `client.` methods), which can be helpful in understanding how it works and how it can be used.
```